### DISCUSSION TITLE
export all table to xslx, with headers and sticky and getter

### DISCUSSION THREAD

**Discussion by Alkimay** [nick-keller/react-datasheet-grid#152] [Feature requests]
is it possible to export all table to xslx, with headers and sticky and getter

**Comment by nick-keller** [comment]
Hey @Alkimay ,
It sure is possible, but you'd have to use an external package for that. This library is only responsible for displaying the data in an excel-like manner. But since you have that data somewhere in a state, it should be very straightforward to use the tool of your choice. 
I've used [ExcelJs](https://www.npmjs.com/package/exceljs) in the past and was satisfied.
